src/components/tasks/card/card.module.scss

```diff
@@ -97,7 +105,41 @@ $black: #252525;
 .progressYellow {
   background-color: $yellow;
 }
+
 .editButton {
   text-align: left;
   cursor: pointer;
 }
+
+.dropdownMenu {
+  position: absolute;
+  right: 0;
+  top: 90%;
+  z-index: 10;
+}
+
+.inputAssigneeSearch {
+  margin-bottom: .5rem;
+}
+
+.assigneeMenu {
+  border-radius: .25rem;
+  background-color: #ffffff;
```

Comment:
use a variable here instead of hex value so that it is easy to change in one place in the future to change it in other areas as well.

Reply:
ok

Reply:
I made the change, please confirm so that I can close this convo